### ISSUE TITLE
Not working with newer presets

### ISSUE THREAD

**Issue by Poiuy69** [b-vesco/vesco-helixsorter#1]
Could be me, but all I get is an empty folder with presets using the latest from customtone

**Comment by b-vesco** [comment]
@Poiuy69, confirmed. I can reproduce this. Looks like some new fields are gumming up some old parts of the code. There are a lot of strict-schema leftovers from when I was writing a full patch management/editor system (before Line 6 provided one). I'll put this on the books to fix and will try to get to it in time for use with Helix 2.80.0 firmware presets. Of course, I'm always happy to take pull requests if you are a java person :)

```
[main] INFO com.benvesco.vesco_helixsorter.App - processing Brit Plexi Jump.hlx
[main] ERROR com.benvesco.vesco_helixsorter.util.Json - exception while parsing JSON: Unrecognized field "meta" (class com.benvesco.vesco_helixsorter.model.HlxPatchFile), not marked as ignorable (3 known properties: "version", "schema", "data"])
 at [Source: ./Brit Plexi Jump.hlx; line: 510, column: 13] (through reference chain: com.benvesco.vesco_helixsorter.model.HlxPatchFile["meta"])
Exception in thread "main" java.lang.NullPointerException
	at com.benvesco.vesco_helixsorter.App.getUsedModelsFrom(App.java:113)
	at com.benvesco.vesco_helixsorter.App.traverse(App.java:73)
	at com.benvesco.vesco_helixsorter.App.main(App.java:50)```

**Comment by b-vesco** [comment]
This is now fixed by [Release 1.0.4](https://github.com/b-vesco/vesco-helixsorter/releases/tag/1.0.4)
